broadway.js

```diff
@@ -38,30 +38,19 @@ function openPages(urls) {
 		// Open the URL in a new tab/window and push the opened window object into the array
 		opened.push(window.open(url, "_blank"));
 	}
-  return opened;
-}
-
-function appendScript(opened, script) {
-	for (const win of opened) {
-		const scriptEl = win.document.createElement("script");
-    scriptEl.src = script;
-    win.document.head.appendChild(scriptEl);
-	}
+	return opened;
 }
 
 function enterDirectLotteries() {
-  const opened = openPages(directEnter);
-  appendScript(opened, "./directLottery.js");
+	const opened = openPages(directEnter);
```

Comment:
Use 2 spaces instead of tab character for indentation please!

Reply:
No, use 4-space tab instead of spaces for indentation please!

Reply:
4 spaces instead of tab character?

Reply:
Sure